doc/how_to_guides/usage.md

```diff
@@ -33,64 +34,83 @@ Runtime options:
-bundle exec rake onesie:new['MyTask','high'] # creates a high priority task
-bundle exec rake onesie:new['MyTask','high','SampleTemplate'] # creates a high priority task using 'SampleTemplate'
+bundle exec rake onesie:new['MyTask']                   # creates a normal priority task
+bundle exec rake onesie:new['MyTask','high']            # creates a high priority task
+bundle exec rake onesie:new['MyTask',,'SampleTemplate'] # creates a normal priority task using 'SampleTemplate'
+
 ```
 
-## Write your Onesie Task
-The generator will create a onesie template file. The task must include the
-`#run` method. Add your code to the new task file.
+## Write your onesie task
+The generator will create a onesie file for you to adjust and
+populate as needed. Your edits should occur mainly (if not
+exclusively) within the required `#run` method.
 
-For details about the Task class and the class macros, see the
+For details about the task class and the class macros, see the
 [Task](../explanations/task.md) document.
 
-## Run the Onesie Task
-There are three Rake tasks to assist with running onesies.
+## Run the onesie task
+There are three rake tasks to assist with running onesies.
 
 ```bash
-rake onesie:run[filename]              # Manually run a specific Onesie Tasks
-rake onesie:run_all                    # Run all unprocessed Onesie Tasks
+rake onesie:run[filename]              # Manually run a specific onesie task
+rake onesie:run_all                    # Run all unprocessed onesie tasks
 rake onesie:run_tasks[priority_level]  # Run all tasks by priority level
+
 ```
 
 !!! Note
-    Manual tasks must be ran individually with the `onesie:run` rake task.
+    Manual tasks must be run individually with the `onesie:run` rake task. Running `onesie:describe` will helpfully generate the command needed to run each file as part of its listing of unprocessed tasks.
 
 ### Examples
 
 ```bash
 be rake onesie:run['20220105140152_my_task'] # Run 20220105140152_my_task.rb
-be rake onesie:run_tasks['high']             # Run all high priority tasks
-be rake onesie:run_tasks                     # Run all tasks without priority
+be rake onesie:run_tasks['high']             # Run all high-priority tasks
+be rake onesie:run_all                       # Run all unprocessed tasks
+
 ```
 
-## Rerunning a Task
-During the development process, it may be useful to rerun a Onesie Task.
-Use the `onesie:rerun` rake task to:
+## Re-running a task
+During the development process, it may be useful to re-run a task.
+Use the `onesie:rerun` rake task without an argument to re-run the most recent task, or specify a filename to re-run a specific task.
 
-- Rerun the most recent Task
-- Specify a Task filename to rerun a specific Task
+!!! Note
+    As stated in the [onesie overview](../explanations/overview.md), a onesie should be _idempotent_—infinitely re-runnable without generating any errors or redundant data.
```

Comment:
This seems like an odd place to say something we've already said. If you want to make sure people don't miss out on the best practices, maybe include it at the end of the `Generate Onesie` section at the top of this file? Ie. something like, "Please review onesie best practices before writing your onesie."
```suggestion
```

Reply:
Tangents, cross-references, and reminders are all expected contents of a note callout, IMO. This is a tricky point that I feel is worth repeating in the context of the `:rerun` command.